src/Symfony/Component/Dotenv/.env

```diff
@@ -0,0 +1 @@
+FOOBAR=${BAR:=production}
```

Comment:
It looks like this file was committed accidentally.

Reply:
Aah yes, sorry for that.